src/components/UploadDropzone.tsx

```diff
@@ -0,0 +1,207 @@
+      acceptedFileItems.find(
+        (file) => file.name === acceptedFiles.find((f) => f.name === file.name)?.name,
+      ) === undefined
+    ) {
+      acceptedFiles.map((file: File) => setAcceptedFileItems([...acceptedFileItems, file]))
+    }
+    if (
+      fileRejections.length > 0 &&
+      rejectedFileItems.find(
+        ({ file }) => file.name === fileRejections.find((f) => f.file.name === file.name)?.file.name,
+      ) === undefined
+    ) {
+      fileRejections.map(({ file, errors }: FileRejections) =>
+        setRejectedFileItems((prev) => [...prev, { file, errors }]),
+      )
+    }
+  }, [acceptedFiles, fileRejections])
+
+  useEffect(() => {
+    setData(acceptedFileItems)
+  }, [acceptedFileItems, setData])
+
+  function formatFileSize(size: number) {
+    const formattedSize: number = size / 1000
+    if (formattedSize > 1000) return `${(formattedSize / 1000).toFixed(2)} MB`
+
+    return `${formattedSize.toFixed(2)} KB`
+  }
+
+  function getErrorMessage() {
+    if (maxFilesToUpload && fileRejections.length > maxFilesToUpload)
+      return `Too many files selected (maximun files: ${maxFilesToUpload})`
+    if (isOnlyImagesAllowed) return 'upload jpg, png, gif or svg image'
+    return 'upload PDF documents'
+  }
+
+  return (
+    <div>
+      <Container
+        {...getRootProps({ isDragActive, isDragAccept, isDragReject, className: 'dropzone disabled' })}
+      >
+        <input {...getInputProps()} />
+        <P
+          label={translate(
+            `Drag 'n drop some${isOnlyImagesAllowed && ' image'} files here, or click to select files`,
+          )}
+        />
+      </Container>
+      {(acceptedFileItems.length > 0 || rejectedFileItems.length > 0) && (
+        <section className="section">
+          <aside>
+            {acceptedFileItems.length > 0 && (
+              <Section>
+                <h4 style={{ color: 'green' }}>{translate(`Accepted files`)}</h4>
+                {acceptedFileItems.map((file: File) => (
+                  <ListItem key={file.name}>
+                    {isOnlyImagesAllowed && (
+                      <img
+                        src={URL.createObjectURL(file)}
+                        width={40}
+                        height={40}
+                        alt={file.name}
+                        style={{ marginRight: '10px' }}
+                      />
+                    )}
+                    <P label={`${file.name} - ${formatFileSize(file.size)}`} style={{ fontSize: 12 }} />
+                  </ListItem>
+                ))}
+              </Section>
+            )}
+            {rejectedFileItems.length > 0 && (
+              <Section>
+                <h4 style={{ color: 'red' }}>{translate(`Rejected files - ${getErrorMessage()}`)}</h4>
+                {rejectedFileItems.map(({ file, errors }: FileRejections) => (
+                  <ListItem key={file.name}>
+                    <P label={`${file.name}`} style={{ fontSize: 12 }} />
```

Comment:
Here you should add the flag `isLabelTranslated` that it does not pop-up as missing key in the i18next debug mode.